devito/passes/clusters/cse.py

```diff
@@ -70,6 +70,12 @@ def cse(cluster, sregistry=None, options=None, **kwargs):
     min_cost = options['cse-min-cost']
     mode = options['cse-algo']
 
+    # Unsafe, and almost always uselsss as these tend to be no-op Clusters, to
```

Comment:
useless

Reply:
agreed, dropping

Reply:
I actually meant the typo: uselsss, but happy to help indirectly